pages/form.js

```diff
@@ -0,0 +1,162 @@
+import Navbar from '../components/Navbar';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import StyledButton from '../components/Button';
+import Title from '../components/Header';
+import { today } from '../utils/date';
+import { getAgeGroup } from '../utils/extract';
+import { useRouter } from 'next/router';
+
+export default function FormPage() {
+  const router = useRouter();
+  const [title, setTitle] = useState('');
+
+
+  function handleTitleChange(event) {
+    const newTitle = event.target.value.trimStart();
+
+    setTitle(newTitle);
+  }
+
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+
+    const form = event.target;
+    const title = form.title.value.trimEnd();
+    const location = form.locations.value;
+    const activity = form.activities.value;
+    const ageGroup = getAgeGroup([...form.ageGroup]);
+    const description = form.description.value;
+    const calendar = form.calendar.value;
+
+
+    const newMeeting = {
+      title,
+      location,
+      activity,
+      ageGroup,
+      description,
+      calendar,
+      id: nanoid(),
+    };
+
+    updateLocalStorage(newMeeting);
+    router.push('/meetings');
+  }
+  function updateLocalStorage(newMeeting) {
+
+    const allMeetings = JSON.parse(localStorage.getItem('meetings')) || [];
+
+    allMeetings.push(newMeeting);
+
+    localStorage.setItem('meetings', JSON.stringify(allMeetings));
+  }
+  return (
+    <>
+      <Title>Erstelle deinen Treffpunkt</Title>
+
+      <form onSubmit={handleSubmit}>
+        <section>
+          <div>
+            <label>Titel</label>
+            <input
```

Comment:
For each `<label>` that you use, you need an `id` attribute on your input/textarea/etc. :)